prow/cmd/checkconfig/main.go

```diff
@@ -793,6 +847,84 @@ func validateNeedsOkToTestLabel(cfg *config.Config) error {
 	return errorutil.NewAggregate(queryErrors...)
 }
 
+type filePresenceChecker interface {
+	remoteCheck(path string) (bool, error)
+	localCheck(path string, client *git.Client) bool
+}
+
+type _filePresenceChecker string
+
+func (f _filePresenceChecker) remoteCheck(path string) (bool, error) {
+	blobUrl := fmt.Sprintf("https://github.com/%s/blob/master/%s", f, path)
```

Comment:
We should add this to the GitHub client

Reply:
Yeah, was thinking about that too, was not sure if "making URLs" is really something a GH API client should do.